api/config/swagger.py

```diff
@@ -15,7 +16,7 @@
 def schema_view_swagger(request):
     schema = coreapi.Document(
         title='Questioner API',
-        url='https://questioners-two-staging.herokuapp.com/',  # replace with your url
+        url=os.getenv('swagger_url'), 
```

Comment:
Hello @IssaIan ,
By convention, environment variables are usually defined using all capital letters. This helps users distinguish environmental variables within other contexts.
You can change the variable `sawagger_url` to `SWAGGER_URL` to meet that convention.

Reply:
Implemented